packages/react-app/pages/profile/edit-profile.js

```diff
@@ -9,6 +19,8 @@ import modelAliases from "../../model.json";
 import { ceramicCoreFactory, CERAMIC_TESTNET, CERAMIC_TESTNET_NODE_URL } from "../../ceramic";
 import { Web3Context } from "../../helpers/Web3Context";
 
+const emojis = ["🚀", "🙏", "👻", "🤌", "👀"];
```

Comment:
Can you use this array of emoji instead ? https://gist.github.com/anthonydelgado/528d1fab9242067348c0ac25f873d7f0